basic_vpc_network_peering/main.tf

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+provider "null" {
+  version = "~> 2.1"
+}
+
+provider "google" {
+  version = "~> 3.45.0"
+}
+
+# [START vpc_peering_create]
```

Comment:
With the module, you only have two do the peering once. It creates both ends (bi-directional).